poutyne/framework/model.py

```diff
@@ -27,11 +27,15 @@ class Model:
 
     Args:
         model (torch.nn.Module): A PyTorch module.
-        optimizer (torch.optim.Optimizer): Initialized PyTorch optimizer.
-        loss_function: Loss function. It can be any PyTorch loss layer or custom loss function. It
-            can also be a string with the same name as a PyTorch loss function (either the functional or
-            object name). The loss function must have the signature ``loss_function(input, target)``
-            where ``input`` is the prediction of the network and ``target`` is the ground truth.
+        optimizer (Union[torch.optim.Optimizer, str]): If torch.optim.Optimier, an initialized PyTorch.
+            If str, should be the optimizer's name in Pytorch (i.e. 'Adam' for torch.optim.Adam).
+            (Default value = 'sgd')
+        loss_function(Union[Callable, str]) It can be any PyTorch
```

Comment:
There is a line return too soon here.

Reply:
Fixed by 824baebe32518051303b48ca9196d19192ee4578